docs/user-guide/export-data.md

```diff
@@ -1,16 +1,19 @@
 # Export Layer Data
+
 **************************
 
-[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to export both vector and raster layers present in  [TOC](toc.md#table-of-contents). In order to provide advanced export capabilities the[ WPS Download process](https://docs.geoserver.org/stable/en/user/community/wps-download/index.html) must be installed and available in GeoServer. MapStore performs a preventive check for this as soon as the user opens the tool: if the WPS Download process is not available, MapStore uses the WFS service as fallback and the export options are limited (eg. only vector data can be exported). 
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to export both vector and raster layers present in  [TOC](toc.md#table-of-contents). In order to provide advanced export capabilities the[WPS Download process](https://docs.geoserver.org/stable/en/user/community/wps-download/index.html) must be installed and available in GeoServer. MapStore performs a preventive check for this as soon as the user opens the tool: if the WPS Download process is not available, MapStore uses the WFS service as fallback and the export options are limited (eg. only vector data can be exported).
```

Comment:
[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to export both vector and raster layers present in  [TOC](toc.md#table-of-contents). In order to provide advanced export capabilities the [WPS Download process](https://docs.geoserver.org/stable/en/user/community/wps-download/index.html) must be installed and available in GeoServer. MapStore performs a preventive check for this as soon as the user opens the tool: if the WPS Download process is not available, MapStore uses the WFS service as fallback and the export options are limited (eg. only vector data can be exported).